README.md

```diff
@@ -10,20 +17,27 @@ while returning a variable length output, and vice versa.
 
 Rubato can be used in realtime applications without any allocation during
 processing by preallocating a [Resampler] and using its
-[input_buffer_allocate](Resampler::input_buffer_allocate) and
-[output_buffer_allocate](Resampler::output_buffer_allocate) methods before
-beginning processing. The [log feature](#log-enable-logging) feature should be disabled
+[process_into_buffer](Resampler::process_into_buffer) and
+method to process into apre-allocated output buffer.
```

Comment:
Not sure why the link to `output_buffer_allocate` was removed? This sentence doesn't make sense anymore.

Reply:
Thanks, looks like I didn't finish editing the sentence.
The _allocate methods are gone since I don't think they make much sense now that input and output can be so many other things than vecs of vecs.